.pre-commit-config.yaml

```
@@ -1,3 +1,5 @@
+exclude: '(.*\.fits(\.fz)?)|(gammapy/catalog/tests/\/data\/.*$)'
```

I run it locally with this exclusion and it was still including those catalog files. This excludes the entire folder which only contains the txt files and I think it is simpler:
```suggestion
exclude: '(.*\.fits(\.fz)?)|(gammapy/catalog/tests/data/)'
```